index.js

```diff
@@ -1,17 +1,73 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    let isInserted = false;
+    for (let i = 0; i < this.length; i++) {
+      if (item <= this.items[i]) {
+        this.items.splice(i, 1, item, this.items[i]);
+        isInserted = true;
+        break;
+      }
+    }
+    if (isInserted === false) {
+      this.items.push(item);
+    }
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.length) {
+      throw new Error("OutofBounds");
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items[this.length - 1];
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items[0];
+    }
+  }
 
-  sum() {}
+  sum() {
+    return this.length === 0
+      ? 0
+      : this.items.reduce((acc, item) => {
```

Comment:
Legal!!! Ternário é show de bola hehehe

Nesse caso, vc tb poderia retornar direto o reduce. Basta passar para ele um ZERO como segundo argumento, assim:

return this.items.reduce((acc, item) => acc + item, 0);

Esse ZERO é o valor inicial do seu acumulador! Se seu array estiver vazio, o reduce vai retornar esse ZERO e fim hehehe